src/role-access/containers/rejected-request-view/rejected-request-view.component.spec.ts

```diff
@@ -85,6 +86,25 @@ describe('RejectedRequestViewComponent', () => {
     expect(router.navigate).toHaveBeenCalledWith([`/cases/case-details/123456789/specific-access-request`]);
   });
 
+  it('should show default message if infoRequired is false', () => {
+    expect(component.reviewReason).toEqual('No reason for rejection found');
+  });
+
+  it('should not show infoRequiredComment if infoRequired is false and infoRequiredComment', () => {
+    const reviewReason = component.getRejectReason(false, 'Need more Info');
+    expect(reviewReason).toEqual(RejectionReasonText.Rejected);
+  });
+
+  it('should not show infoRequiredComment if infoRequired is true and infoRequiredComment is empty', () => {
+    const reviewReason = component.getRejectReason(true, null);
+    expect(reviewReason).toEqual(RejectionReasonText.MoreInformation);
+  });
+
+  it('should show infoRequiredComment if infoRequired is true and infoRequiredComment', () => {
+    const reviewReason = component.getRejectReason(true, 'Need more Info');
+    expect(reviewReason).toEqual('Need more Info');
+  });
```

Comment:
Tests are good but made me think we do not need infoRequired field itself now - the second scenario should never be an option

Reply:
InfoRequiredComment can be empty. So, InfoRequired is required